manifests/profile/master/postgres_access.pp

```diff
@@ -0,0 +1,63 @@
+# @summary Apply this class to a PE-managed postgres instance to allow access from telegraf
+#
+# @param telegraf_host
+#   The FQDN of the host where telegraf runs.  
+#   Defaults to an empty string.  You can explicitly set this parameter or the class attempts to lookup which host has the puppet_metrics_dashboard class applied in PuppetDB.  If the parameter is not set and the lookup does not return anything we issue a warning.
+#
+class puppet_metrics_dashboard::profile::master::postgres_access (
+  String $telegraf_host = ''
```

Comment:
I thought you were running Telegraf locally and shipping the metrics? If that is the case then this is unneeded and the code below should be updated to reflect that telegraf is on the local machine.